Sources/DatabaseKit/ConnectionPool/DatabaseConnectionPoolConfig.swift

```diff
@@ -9,20 +7,20 @@ public struct DatabaseConnectionPoolConfig: ServiceType {
     ///
     /// The minimum supported value is `1`, meaning that your database must be able to handle
     /// at least n connections where n = num logical CPU cores.
-    public var maxConnections: UInt
+    public var maxConnections: Int
```

Comment:
Shouldn’t this be constant?

Reply:
It's a `struct` and defined as `let` on both the cache and pool. Having it a var here just makes it easier to configure:

```swift
var config = DatabaseConnectionPoolConfig()
config.maxConnections = 20
services.register(config)
```